src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -185,10 +185,12 @@ def create_conversation(client: WebClient, name: str, is_private: bool = False)
         is_private=is_private,
     )["channel"]
 
+    domain = make_call(client, SlackAPIGetEndpoints.team_info)["team"]["domain"]
```

Comment:
Same with this one if we needed to make the call in more than one place. Might be fine to leave it as is for now.

Reply:
Acknowledged. I'll keep it in for now.